_common/README.md

```diff
@@ -6,3 +6,53 @@ deployed to keyman sites automatically as part of bootstrap.inc.sh.
 ## Namespace
 
 The root namespace for all PHP modules is `Keyman\Site\Common`.
+
+# GFMAlerts (GitHub Flavored Markdown Alerts)
+
+Here are the syntaxes of the Alerts for Keyman websites. There are 5 types of GFMAlerts:
+![GFMAlerts](./assets/img/gfmalerts.png)
```

Comment:
https://github.com/keymanapp/shared-sites/blob/ad297a4c0fc4786086ee7880e4e568fdbd63b0e5/bootstrap.inc.sh#L95-L132

We may need to update  to also download the new .png file.
Since the website will pull down this README, the link-checkers would fail if the assets img doesn't exist.

Reply:
@darcywong00, I rebased this to alerts/v0.12. Can this be merge for testing?